webpack.config.js

```diff
@@ -252,9 +258,9 @@ if ( process.env.DASHBOARD ) {
 	webpackConfig.plugins.unshift( new DashboardPlugin() );
 }
 
-if ( process.env.WEBPACK_OUTPUT_JSON ) {
+if ( process.env.WEBPACK_OUTPUT_JSON || process.env.NODE_ENV === 'production' ) {
 	webpackConfig.devtool = 'cheap-module-source-map';
```

Comment:
I actually think that with uglify on we need to turn on uglify's source maps instead